custom/icds_reports/utils/vhsnd_datapull/format_vhsnd_Data.py

```diff
@@ -0,0 +1,47 @@
+import csv
```

Comment:
Why is this in utils? This isn't really a utility, which is ideally for shared functionality used in a bunch of places

Reply:
also, how is this executed? If it is just a random script it should be a management command. as it is now, it is executed every time hq is started. this should probably go in a function, especially if it is going to be used in the future, which it sounds like

Reply:
Hey this script is not to be added to management command or to be merged. This python code is to just format the output csv we get from the sql command. and this python will be run  on local not on any server.

Reply:
so how will you run this? 
Also, if we just resourced someone on adding this to hq (which we did), it would be way better to have this code already in HQ rather than in a weird unmergeable state. It shouldn't take too long to just wrap it in a single function

Reply:
Yeah, that work of adding it to hq is for next month when this comes which we decided late today. But this came to me in the evening so it was not enough time to add this to hq and deploy it and then run it. Because this data is needed tomorrow morning.

Reply:
Yes I will run this on my local. I am keeping this here in this PR so that Challa can refer this while adding it to HQ page.